src/injections/js/bug1836157-thai-masszazs-niceScroll-disable.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Please add a license header like

https://github.com/mozilla-extensions/webcompat-addon/blob/cdc9edd6858a9713a39d0e441afdce1a1647f721/src/injections/js/bug0000000-testbed-js-injection.js#L1-L3

to this intervention and all others as well (including CSS).